kde-apps/kmime/kmime_kf6-25.04.3.recipe

```diff
@@ -40,7 +37,7 @@ REQUIRES_devel="
 ARCHITECTURES_doc="any"
 
 PROVIDES_doc="
-	kmime_kf6${secondaryArchSuffix}_doc = $portVersion
+	kmime_kf6_doc = $portVersion
 	"
 REQUIRES_doc="
 	kmime_kf6${secondaryArchSuffix}_devel == $portVersion base
```

Comment:
The "any" doc package can't use $secondaryArchSuffix in its metadata, otherwise it would be different for primary and secondary architectures. I would just remove this line (see e.g. this: https://github.com/haikuports/haikuports/commit/d5554de18aa17d8a50a8b4d2005216b75824317c#diff-93ba69c436735dfc5abd696217944847feb8eb649d8baa062b1f96deb9c0387fL100)

Also, I would add `REPLACES_doc="kmime_kf6_x86_doc"` (unconditionally) to facilitate upgrades.

Reply:
Ah, almost misread this :) yeah makes sense to not rely on any $secondaryArchSuffix.

In the latter case it's obvious also to add the REPLACES. Thanks for reviewing!